app/components/NoticeTypeCheckboxes/NoticeTypeCheckboxes.tsx

```diff
@@ -196,7 +197,6 @@ const JsonNoticeTypes: { [key: string]: string[] } = {
   LVK: ['igwn.gwalert'],
   Swift: ['gcn.notices.swift.bat.guano'],
   'Einstein Probe': ['gcn.notices.einstein_probe.wxt.alert'],
-  SVOM: ['gcn.notices.svom'],
```

Comment:
Instead:

You can conditionally populate this field using the ternary `?:` operator and object spread syntax `...`:

```ts
  'Einstein Probe': ['gcn.notices.einstein_probe.wxt.alert'],
  ...(feature('SVOM_QUICKSTART') ? {SVOM: ['gcn.notices.svom']} : {}),
```

Reply:
Apologies for the delayed reply; I tried the conditional population method prior to the current commit, but it resulted in the quickstart becoming unresponsive beyond step 2 as described [here](https://github.com/nasa-gcn/gcn.nasa.gov/pull/2404#issuecomment-2187393308)

Reply:
Actually I see that the way you did it is the way that we have done it in the past.